handbook/engineering/web/onboarding.md

```diff
@@ -0,0 +1,89 @@
+
+Never assume that a problem is somebody else's to fix!
+
+## Getting set up
+
+You'll have to get some basics set up in your first few days:
+- [Configure your GitHub notifications.](../github-notifications/index.md)
+- Join #dev-announce, #dev-chat, and your team's channel on Slack. [Team chat documentation](../communication/team_chat.md#engineering)
+- Set up your [local development environment](https://github.com/sourcegraph/sourcegraph/blob/master/doc/dev/local_development.md#step-1-install-dependencies). If you encounter any issues, ask for help in Slack and then update the documentation to reflect the resolution (so the next engineer that we hire doesn't run into the same problem).
+- Read through your team's [handbook page](../index.md), to learn about the team and its internal processes.
+
+## Weeks 1-3
+
+### Starter tasks
+
+Your manager will assign to you three starter tasks that you should aim to complete in your first two weeks. These tasks are small, atomic, and intended to expose you to different parts of our codebase and product: it's important that you build the flexibility to fix any problem you'll be faced with at Sourcegraph, and don't narrow down your comfort zone to a narrow part of our codebase and product.
+
+As you're working on these tasks:
+- Aim to be as incremental as possible:
+    - Open a pull request as soon as you feel like you're ready for feedback or input on your code -- you can make it a draft pull request if your code is still a work in progress.  
+    - Favour splitting up your work in multiple pull requests every time it makes sense -- shipping frequently is important.
+    - Ask yourself what tests are appropriate for the change you're tackling, and add them!
+- Communicate on your progress with the team through our [daily Slack updates](../index.md#daily_slack_updates).
+- If you need help, remember that there are [no stupid questions](#there_are_no_stupid_questions) -- ask for help in your team's channel (or any appropriate channel), and add the answer to our docs or the handbook if you feel like it can help future teammates.
+
+As you complete these tasks, share your accomplishments in #progress 🙂
+
+### Pairing sessions
+
+Reach out to every member of your direct team, and set up a two-hour pairing session with them. These sessions will be an opportunity to get to know your teammates, understand what they're working on and why, and learn more about our codebase and development flows!
+
+Take the first 20-30 minutes of the session to have an unstructured, introductory chat. Then, start hacking! Your teammate will set up a [live share](https://visualstudio.microsoft.com/services/live-share/) and walk you through what they're currently working on. Ask as many questions as possible, to try to understand:
+- What problem is your teammate trying to solve?
+- Why is that problem important? What team goal does it fit under?
+- Was are relevant RFCs, PDs, GitHub issues or documentation pages?
```

Comment:
I anticipate fresh hires won't know what these mean otherwise :)
```suggestion
- Was are relevant [RFCs](https://about.sourcegraph.com/handbook/communication/rfcs), [PDs](https://about.sourcegraph.com/handbook/product/product_documents), GitHub issues or documentation pages?
```

Reply:
Ah thanks, I meant to add these but forgot to update at the end 🙂